### PR TITLE
PodHasNetwork condition blog

### PR DESCRIPTION
**Pull request by ddebroy**
Placeholder for PodHasNetwork condition blog [https://github.com/kubernetes/enhancements/issues/3085]

### CODE REVIEW

**Comment by cathchu** [comment]
Hey there 👋 @ddebroy , 1.25 Release Docs Shadow here!

Thanks for filing this placeholder. As a reminder, final docs PRs are due August 9th. Take a look at [Documenting a release](https://kubernetes.io/docs/contribute/new-content/new-features/#for-developers-or-other-sig-members) for additional information for the docs requirement for the release.

Thanks again!

**Comment by katcosgrove** [comment]
Hi from the Comms team! Just a reminder that the Ready to Review deadline for feature blogs is Tuesday, August 16. You will also be assigned a publication date post-release. Is there anything we can do to help you right now?

**Comment by ddebroy** [comment]
@katcosgrove this is now ready for review.

**Comment by Sea-n** [comment]
Remove labels introduced by changing base branch.
/remove-area release-eng
/remove-language de fr ja ko pt zh

File: content/en/blog/_posts/2022-08-14-pod-has-network-condition.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
+---
+layout: blog
+title: 'Kubernetes 1.25: PodHasNetwork condition for pods'
```
Comment (Garima-Negi) [inline content/en/blog/_posts/2022-08-14-pod-has-network-condition.md:3]: Should this be `Kubernetes v1.25` instead?

File: content/en/blog/_posts/2022-08-14-pod-has-network-condition.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+date: 2022-05-27
+slug: pod-has-network-condition
+---
+
+**Author:**
+Deep Debroy (Apple)
+
+Kubernetes 1.24 introduces Alpha support for a new kubelet managed pod condition
```
Comment (Garima-Negi) [inline content/en/blog/_posts/2022-08-14-pod-has-network-condition.md:11]: similar to previous comment, `v1.24` would be more appropriate

Comment (Garima-Negi) [inline content/en/blog/_posts/2022-08-14-pod-has-network-condition.md:11]: similar to previous comment, v1.24 would be more appropriate

**Review by Garima-Negi — CHANGES_REQUESTED** [review]

File: content/en/blog/_posts/2022-08-14-pod-has-network-condition.md (hunk 2 above)
Comment (ddebroy) [reply content/en/blog/_posts/2022-08-14-pod-has-network-condition.md:11]: good catch - fixed!

File: content/en/blog/_posts/2022-08-14-pod-has-network-condition.md (hunk 1 above)
Comment (ddebroy) [reply content/en/blog/_posts/2022-08-14-pod-has-network-condition.md:3]: `Kubernetes v1.25` is accurate. The instance below in the text was incorrectly referring to `1.24` which is fixed.

**Comment by katcosgrove** [comment]
Hi from Comms! Your assigned publication date is September 14. Thank you!

File: content/en/blog/_posts/2022-08-14-pod-has-network-condition.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
+---
+layout: blog
+title: 'Kubernetes 1.25: PodHasNetwork condition for pods'
+date: 2022-05-27
```
Comment (sftim) [inline content/en/blog/_posts/2022-08-14-pod-has-network-condition.md:4]: Please update this field _and_ rename the file, to match the assigned publication date: 2022-09-14.

**Review by sftim — DISMISSED** [review]

File: content/en/blog/_posts/2022-08-14-pod-has-network-condition.md (hunk 3 above)
Comment (ddebroy) [reply content/en/blog/_posts/2022-08-14-pod-has-network-condition.md]: done

**Comment by ddebroy** [comment]
superseded by https://github.com/kubernetes/website/pull/36197

**Pull request closed without merging**